src/components/post/index.js

```diff
@@ -81,33 +103,30 @@ const Post = ({
 
         <section
           className={`post-interactions-container border-top ${
```

Comment:
Can we extract this out to before the `return` and create a variable for the classes. I think it's a bit difficult to read what we're doing here